### ISSUE TITLE
Improvement on rendering

### ISSUE THREAD

**Issue by r0ms** [timwah/pokeslack#25]
Hey !
Glad we had the same idea ! I had forked the pokemap repo to create our own notifier on our slack channel. I'm a python guru but I managed to have a great rendering of the notification. 
Maybe you could have a look on it ;)
![image](https://cloud.githubusercontent.com/assets/1615795/17265894/2330d05a-55f3-11e6-8aa5-449df488f402.png)
Here is what it give on mobile ;)

**Comment by timwah** [comment]
Awesome! Thanks for sharing, it looks awesome. Beautifying our rendering is definitely in the pipeline and this will be a great starting point. Btw, if you are forked off the PokemonGo-Map repo, I suggest trying this implementation instead (and contributing your visual updates 😄 ). Reason being is that this is a much slimmer implementation off of the [PGOAPI](https://github.com/tejado/pgoapi), and we have the latest bug fixes as well (see #23 #26); they seem to move pretty slowly over there and not sure when you'll get these changes. Cheers!

**Comment by r0ms** [comment]
Yup, I'll definetely try this implementation. In the future, I think it could be great to have multiple scanner on multiple location and different type of notifications (slack room, pushbullet, mail, hipchat, etc.).
As I said, I'm not a python guru but it's great way to learn a new language :)

**Comment by r0ms** [comment]
Hey @timwah !
Didn't have a lot of news those last days. Did you have a check on my PR ?
